packages/terra-application-navigation/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Changelog
 Unreleased
 ----------
 ### Fixed
+* Fixed Drawer menu not closing in Internet Explorer on clicking outside of drawer menu.
+
+### Changed
+* Updated `content-layout` background color.
```

Comment:
has `content-layout` background color changed....??

Reply:
This is from a change that has since been reverted. We should remove this.

Reply:
Removed here 17a0ea9012a2a5806429e85c6939eb0a35cdbbdb and here d6e207cf53358ec57af360c2959bc6588b27b481